tests/key_field/key_field_sample_generated.h

```diff
@@ -67,12 +97,12 @@ FLATBUFFERS_MANUALLY_ALIGNED_STRUCT(1) Baz FLATBUFFERS_FINAL_CLASS {
   bool KeyCompareLessThan(const Baz * const o) const {
     return KeyCompareWithValue(o->a()) < 0;
   }
-  int KeyCompareWithValue(const ::flatbuffers::Array<uint8_t, 4> *_a) const {
-    const ::flatbuffers::Array<uint8_t, 4> *curr_a = a();
-    for (::flatbuffers::uoffset_t i = 0; i < curr_a->size(); i++) {
+  int KeyCompareWithValue(const flatbuffers::Array<uint8_t, 4> *_a) const {
```

Comment:
Global scoped, According to this commit : https://github.com/google/flatbuffers/pull/7764